src/pages/docs/worker-bridge.mdx

```diff
@@ -0,0 +1,452 @@
+## Worker Bridge
+
+Worker-Bridge, an integral component of the `Golem OSS` worker-service, empowers you to define and upload API definitions.
+These definitions allow you to expose endpoints to users and  specify how incoming requests should be managed by your `Golem` worker workflow/application.
```

Comment:
'to users' might imply that users don't need an API Gateway. However, we don't really provide authentication or any other useful features, so really this is an internal API. Making a public-facing API would require other things.